src/Native/FactorizationMachineNative/FactorizationMachineCore.cpp

```diff
@@ -48,7 +48,7 @@ EXPORT_API(void) CalculateIntermediateVariablesNative(int fieldCount, int latent
 
         for (int k = 0; k + 4 <= d; k += 4)
         {
-            const __m128 _v = _mm_load_ps(vjf + k);
+            const __m128 _v = _mm_loadu_ps(vjf + k);
```

Comment:
This is a major change to Factorization Machine. Do you have any perf result for this new function?

Reply:
I currently don't have any perf results for this, but I will test it.

Reply:
I'd suggest using Criteo data set which was used when designing this algorithm.

Reply:
do you have any rsp or test for this dataset ?

Reply:
For this PR i reverted the change to not editing the Factorization Machine

Reply:
Factorization machine still uses aligned memory. Can we move forward without touching it?

Reply:
@eerhardt @danmosemsft is it okay to keep the just aligned array for factorization machine ?

Reply:
If we prove it provides value for FactorizaitonMachine, then yes, let's keep it. But move it into the FactorizationMachine assembly.

Reply:
I have already moved it to FactorizationMachine assembly. The main reason for me to skip is that these algorithms are not as straight forward as cpumathnative functions. Changing them will require a lot of time and a person who understands them completely.